### DISCUSSION TITLE
How to exit Read-Only ?

### DISCUSSION THREAD

**Discussion by L-Clem** [gridaco/grida#44] [Q&A] [answered]
Hi just to ask if it is possible to exit the read-only editor?
Thanks in advance 😄

**Comment by softmarshmallow** [comment, marked as answer]
That's not possible for now. We're trying to make surf as a PR managable editor, and we are working on it.
We'll let you updated when this feature is available. thanks for your idea though😀

**Reply by L-Clem** [reply]
👌
